app/models/course.rb

```diff
@@ -446,6 +446,12 @@ def funding_type=(funding_type)
     end
   end
 
+  def fixup_languages
```

Comment:
Is this a one-time method?

Reply:
It will be used whenever `mcb courses create` is used to add a language, and in future could be used in the self-service new course flow

Reply:
Maybe a more descriptive name than `fixup_languages` such as `imply_modern_language` or something?

Reply:
`ensure_modern_languages`
?

maybe we should override assignment instead

Reply:
Thanks @faissalMT